ecosystem/typescript/sdk/packages/aptosClient/src/index.node.ts

```diff
@@ -0,0 +1,107 @@
+import got, { OptionsOfJSONResponseBody, RequestError, Response } from "got";
+import { CookieJar } from "./cookieJar";
+import { AptosClientRequest, AptosClientResponse } from "./types";
+
+const cookieJar = new CookieJar();
+
+export default async function aptosClient<Res>(requestOptions: AptosClientRequest): Promise<AptosClientResponse<Res>> {
```

Comment:
maybe just a comment here that explains what `aptosClient` does, since there are a lot of client names in the SDK

Reply:
haha sure